ibm_mq/tests/conftest.py

```diff
@@ -0,0 +1,89 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import pytest
+import subprocess
+import copy
+import docker
+import logging
+
+from datadog_checks.dev import docker_run  # , temp_dir
+from datadog_checks.ibm_mq import IbmMqCheck
+
+from . import common
+
+log = logging.getLogger(__file__)
+
+
+@pytest.fixture
+def check():
+    return IbmMqCheck('ibm_mq', {}, {})
+
+
+@pytest.fixture
+def instance():
+    inst = copy.deepcopy(common.INSTANCE)
+    container = subprocess.check_output([
+        'docker',
+        'ps',
+        '-q'
+    ])
+    container = container.decode("utf-8").strip()
+
+    inst['docker_container'] = container
+
+    log.warning(container)
+    client = docker.from_env()
+    cont = client.containers.get(container)
+    log.warning(cont.exec_run(" echo hi"))
+
+    return inst
+
+
+@pytest.fixture
+def seed_data():
+    publish = os.path.join(common.HERE, 'python', 'publish.py')
+    consume = os.path.join(common.HERE, 'python', 'consume.py')
+
+    env = copy.deepcopy(os.environ)
+
+    os.environ['HOST'] = common.HOST
+    os.environ['PORT'] = common.PORT
+    os.environ['USERNAME'] = common.USERNAME
+    os.environ['PASSWORD'] = common.PASSWORD
+    os.environ['CHANNEL'] = common.CHANNEL
+    os.environ['QUEUE_MANAGER'] = common.QUEUE_MANAGER
+    os.environ['QUEUE'] = common.QUEUE
+
+    subprocess.check_call(['python', publish])
+    subprocess.check_call(['python', consume])
+
+    # reset the environment
+    os.environ = env
+
+
+@pytest.fixture(scope='session')
+def spin_up_ibmmq():
+
+    if common.MQ_VERSION == '9':
+        log_pattern = "AMQ5026I: The listener 'DEV.LISTENER.TCP' has started. ProcessId"
+    elif common.MQ_VERSION == '8':
+        log_pattern = r".*QMNAME\(datadog\)\s*STATUS\(Running\).*"
+
+    def down():
+        pass
```

Comment:
Why are we not spinning env down?

Reply:
I removed this, it was stale from my testing